src/styles/blocks/card.scss

```diff
@@ -0,0 +1,115 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
```

Comment:
That's all not part of the card block, so shouldn't be here according to BEM-rules (should be in a separate file)